lib/cequel/record/tasks.rb

```diff
@@ -63,12 +63,15 @@ def drop!
     puts "Dropped keyspace #{Cequel::Record.connection.name}"
   end
 
+  def models_dir_path
+    ENV['CEQUEL_MODELS_PATH'] || "#{File.expand_path('app/models', project_root)}/"
```

Comment:
Not sure I get what you're saying on this one. This is operational for me using the CEQUEL_MODELS_PATH to point to an alternative location for the model files.  Help me understand and I'm happy to patch.